lib/qmp.ml

```diff
@@ -160,7 +164,9 @@ let json_of_message = function
       | Eject (device, None) -> "eject", [ "device", `String device ]
       | Eject (device, Some force) -> "eject", [ "device", `String device; "force", `Bool force ]
       | Change (device, target, None) -> "change", [ "device", `String device; "target", `String target ]
-      | Change (device, target, Some arg) -> "change", [ "device", `String device; "target", `String target; "arg", `String arg ] in
+      | Change (device, target, Some arg) -> "change", [ "device", `String device; "target", `String target; "arg", `String arg ]
+      | Xen_save_devices_state filename -> "xen-save-devices-state", [ "filename", `String filename]
+      | Xen_load_devices_state filename -> "xen-load-devices-state", [ "filename", `String filename] in
```

Comment:
If you put `in` in a new line, the next time you add a feature you will avoid two diffs line (as for `Change...`)